src/schemas/config-1.0.0.schema.json

```diff
@@ -74,92 +74,6 @@
         }
       }
     },
-    "annDataCellSets": {
```

Comment:
What happened here?

Reply:
The validation of the schema got delegated to the loader.